android/src/com/eegeo/mobilesdkharness/BackgroundThreadActivity.java

```diff
@@ -220,9 +297,19 @@ public void run()
 						
 						if(deltaSeconds > m_frameThrottleDelaySeconds)
 						{
-							if(m_running)
+							if(m_running && m_headTracker != null)
 							{
-								NativeJniCalls.updateNativeCode(deltaSeconds);
+								float[] tempHeadTransform = new float[16];
```

Comment:
As per early comment this logic is a good candidate to be moved out of BackgroundThreadActivity.